roles/postgresql/tasks/postgresql_yum6.yml

```diff
@@ -0,0 +1,53 @@
+---
+- name: yum |  Get pgdg-centos94-9.4-2 for {{ ansible_distribution }}
+  become: yes
+  yum:
+    name: http://yum.postgresql.org/9.4/redhat/rhel-6-x86_64/pgdg-centos94-9.4-2.noarch.rpm
+    state: installed
+    update_cache: yes
+  when: ansible_distribution == "CentOS"
+  tags: [db, postgresql]
+
+- name: yum | Get pgdg-centos94-9.4-2 for {{ ansible_distribution }}
```

Comment:
This might be confusing. Probably it means "Setup PostgreSQL repository" or something like that.